configs/sandpack-contents/tutorial/package.ts

```diff
@@ -0,0 +1,39 @@
+export const packageJson = `
+{
+  "name": "react",
+  "version": "1.0.0",
+  "description": "React example starter project",
+  "keywords": [
+    "react",
+    "starter"
+  ],
+  "main": "src/index.js",
+  "dependencies": {
+    "@chakra-ui/react": "1.8.6",
+    "@emotion/react": "11.8.1",
+    "@emotion/styled": "11.8.1",
+    "framer-motion": "6.2.8",
+    "hash-it": "5.0.2",
+    "react": "17.0.2",
+    "react-dom": "17.0.2",
+    "react-jazzicon": "1.0.3",
```

Comment:
Where's the `react-jazzicon` package being used?

Reply:
This was in the initial example for the csb. I'll check and clean it up.